en/faq/deployment-azure-portal.md

```diff
@@ -3,6 +3,51 @@ title: How to deploy on Azure Portal?
 description: How to deploy a Nuxt.js application on Azure Portal?
 ---
 
+## Requirements
+- It is required that you select a backend when setting up the project. Even if you don't need it, or else the site wont start up.
+- The server is running Node 8 or greater
+
+## What if I already have a project without an backend?
+No worries. It is easy to add an express server to an existing project.
+
+Download the index.js file from this gist: https://gist.github.com/dasmikko/bef743ad4400f8b1c1c8c6e494f081d6 and place it inside a folder called `server` in the root of your project.
```

Comment:
```suggestion
Use the following index.js file and place it inside a folder called `server` in the root of your project.
```

Could you add the index.js directly in the MD file?

```js
/**
* Place this file inside the "/server" folder in the root project
*/
const express = require('express')
const consola = require('consola')
const { Nuxt, Builder } = require('nuxt')
const app = express()

// Import and Set Nuxt.js options
const config = require('../nuxt.config.js')
config.dev = process.env.NODE_ENV !== 'production'

async function start () {
  // Init Nuxt.js
  const nuxt = new Nuxt(config)

  const { host, port } = nuxt.options.server

  // Build only in dev mode
  if (config.dev) {
    const builder = new Builder(nuxt)
    await builder.build()
  } else {
    await nuxt.ready()
  }

  // Give nuxt middleware to express
  app.use(nuxt.render)

  // Listen the server
  app.listen(port, host)
  consola.ready({
    message: `Server listening on http://${host}:${port}`,
    badge: true
  })
}
start()
```

Reply:
It's inside it now.